tutorials/export/exporting_for_android.rst

```diff
@@ -100,20 +100,18 @@ There are two types of icons required by Godot:
 
 - **Main Icon:** The "classic" icon. This will be used on all Android versions up to Android 8 (Oreo), exclusive. Must be at least 192×192 px.
 - **Adaptive Icons:** Starting from Android 8 (inclusive), `Adaptive Icons <https://developer.android.com/guide/practices/ui_guidelines/icon_design_adaptive>`_ were introduced. Applications will need to include separate background and foreground icons to have a native look. The user's launcher application will control the icon's animation and masking. Must be at least 432×432 px.
-- **Themed Icons:** Starting from Android 13 (inclusive), Themed Icons were introduced. Applications will need to include a monochrome icon to enable this feature. The user's launcher application will control the icon's theme. Must be at least 432×432 px.
```

Comment:
It's better to keep it with other icons. Just add optional here, e.g. Themed Icons (optional)

Reply:
The problem is that doesn't read well since the section starts with "There are two types of icons required by Godot:"

Reply:
@skyace65 Then we can merge themed icon into Adaptive icon.

Adaptive Icons:
Adaptive icons were introduced in Android 8. Applications need to provide separate background and foreground icons to achieve a native look. Themed icons (optional), enabled by providing a monochrome icon, allow launchers to apply the user's chosen theme to the app icon from Android 13 onwards. The user's launcher application controls the icon's animation, masking, and theming based on user preferences. All icons must be at least 432x432 pixels.